tgstation.dme

```diff
@@ -3406,9 +3406,6 @@
 #include "ModularBungalow\transfervote\code\shuttle.dm"
 #include "ModularBungalow\transfervote\code\transfervote.dm"
 #include "ModularBungalow\transfervote\code\transfervote_config.dm"
-#include "ModularBungalow\Mining\items.dm"
-#include "ModularBungalow\Mining\monster.dm"
-#include "ModularBungalow\Mining\rockplanet.dm"
```

Comment:
???

Reply:
that was duplicate, atleast is what the checks/github say, they are still include in the code